src/apis/index.js

```diff
@@ -227,6 +230,7 @@ export default store => {
     getAnnotationUser,
     getCurrentPageNumber: getCurrentPageNumber(store),
     getPageCount: getPageCount(store),
+    getSelectedPageNumbers: getSelectedPageNumbers(store),
```

Comment:
I updated the code in the API, I renamed it from `getSelectedThumbnailPageIndexes ` to `getSelectedPageNumbers`. I think it make sense for the customer to not have to deal with indexes

Reply:
I think in the API name or the documentation we should mention `thumbnail`, otherwise it's not very clear to me what is `selected`. Let me know what you think about this.